### DISCUSSION TITLE
Boot up then launch  game use Overclock crash

### DISCUSSION THREAD

**Discussion by littlehui** [eduardofilo/RG350_adam_image#231] [Hardware problems?]
I set up OC on emuator game list. 
Next time boot up and lanuch game which i set up OC beforo, it crash. Need reset system to reboot.

**Comment by eduardofilo** [comment]
I'm afraid you lost the silicone lotto. This cannot be considered an issue on either the image or ODbeta. Some machines are simply more sensitive to pushing the processor beyond values considered safe.
